pkgs/top-level/all-packages.nix

```
@@ -21375,7 +21375,11 @@ with pkgs;
 
   hasura-cli = callPackage ../servers/hasura/cli.nix { };
 
-  hbase = callPackage ../servers/hbase {};
+  inherit (callPackage ../servers/hbase {}) hbase_1_7_1 hbase_2_4_11 hbase_3_0_0_a2;
+  hbase1 = hbase_1_7_1;
+  hbase2 = hbase_2_4_11;
+  hbase3 = hbase_3_0_0_a2;
+  hbase = hbase2;
```

```suggestion
  inherit (callPackage ../servers/hbase {}) hbase_1_7 hbase_2_4 hbase_3_0;
  hbase_1 = hbase_1_7;
  hbase_2 = hbase_2_4;
  hbase_3 = hbase_3_0;
  hbase = hbase_2;
```

Might as well move hbase to aliases.

---

I've dropped the maintenance versions from package names. But as mentioned above, moving it to aliases just makes updating packages tedious.